lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -128,6 +130,14 @@ XX(down, string, none, down, __VA_ARGS__)
 XX(identifier, string, none, id, __VA_ARGS__) \
 XX(command, TunnelCommand, none, command, __VA_ARGS__)
 
+#define TUNNEL_SET_LOG_LEVEL(XX, ...) \
+XX(loglevel, string, none, Level, __VA_ARGS__)
+
+#define TUNNEL_TUN_IP_V4(XX, ...) \
+XX(tunIP, string, none, TunIPv4, __VA_ARGS__) \
+XX(mask, int, none, TunIPv4Mask, __VA_ARGS__) \
```

Comment:
This field which is called "mask" is actually being used to convey the number of bits in the mask. this is commonly called the "prefix length" .

Reply:
I changed the variable used inside the ZET, but I retained the TunIPv4Mask which gets written into the config file. If we change this now, when we deploy ZET into windows, it will not be able to read this value from the existing TunIPv4Mask value from the config file.

XX(prefixLength, int, none, TunIPv4Mask, __VA_ARGS__) \